pkg/resources/fluentd/prometheusrules.go

```diff
@@ -120,34 +120,6 @@ func (r *Reconciler) prometheusRules() (runtime.Object, reconciler.DesiredState,
 						"description": `Fluentd output error count is "{{ $value }}" for the last 10 minutes.`,
 					},
 				},
-				{
-					Alert: "FluentdBufferSize",
-					Expr:  intstr.FromString(fmt.Sprintf(`node_filesystem_avail_bytes{mountpoint="/buffers", %[1]s} / node_filesystem_size_bytes{mountpoint="/buffers", %[1]s} * 100 < 10`, nsJobLabel)),
-					For:   "10m",
-					Labels: map[string]string{
-						"rulegroup": ruleGroupName,
-						"service":   "fluentd",
-						"severity":  "warning",
-					},
-					Annotations: map[string]string{
-						"summary":     `Fluentd buffer free capacity less than 10%.`,
-						"description": `Fluentd buffer size capacity is {{ $value }}%.`,
-					},
-				},
-				{
-					Alert: "FluentdBufferSize",
-					Expr:  intstr.FromString(fmt.Sprintf(`node_filesystem_avail_bytes{mountpoint="/buffers", %[1]s} / node_filesystem_size_bytes{mountpoint="/buffers", %[1]s} * 100 < 5`, nsJobLabel)),
-					For:   "10m",
-					Labels: map[string]string{
-						"rulegroup": ruleGroupName,
-						"service":   "fluentd",
-						"severity":  "critical",
-					},
-					Annotations: map[string]string{
-						"summary":     `Fluentd buffer free capacity less than 5%.`,
-						"description": `Fluentd buffer size capacity is {{ $value }}%.`,
-					},
-				},
 				{
 					Alert: "FluentdPredictedBufferGrowth",
```

Comment:
Shouldn't this rule be also moved to the buffer volume rule set as this also pertains to the buffer volume?